modules/ROOT/pages/apikit-4-for-odatav2.adoc

```diff
@@ -0,0 +1,17 @@
+= APIkit for OData V2
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+APIkit for OData v2 is a framework that simplifies the development of RESTful APIs that adhere to the OData v2 specification. OData, also called the Open Data Protocol, enables the creation and consumption of RESTful APIs. It allows data querying and manipulation through standard HTTP requests.
+
+APIkit for OData v2 is an extension of APIkit for REST. It builds upon the foundation of APIkit for REST and uses RAML APIs. This integration between APIkit for OData v2 and APIkit for REST combines their functionalities and benefits.
+
+However, APIkit for OData v2 has certain limitations. These limitations can impact its functionality and provide fewer capabilities than newer versions of OData or other frameworks. Consider these limitations when deciding whether APIkit for OData v2 is the appropriate choice for your specific use case.
```

Comment:
Suggest rewording because the text makes it sound like the limitations provide fewer capabilities. 
However, APIkit for OData v2 provides fewer capabilities than newer versions of OData and other frameworks and it has other limitations. such as ...

Reply:
Thanks for the rewording!
Regarding differences between OData v2 and OData v4, they are not easy to list because there are just too many. 
Even OData.org recommends using OData v4 without providing further explanations or details (https://www.odata.org/documentation/odata-version-2-0/).